packages/taquito-utils/test/taquito-utils.spec.ts

```diff
@@ -7,3 +7,14 @@ describe('Encode expr', () => {
     );
   });
 });
+
+describe('String/Bytes conversions', () => {
+  it('Should convert a string to bytes', () => {
+    // I used the result from http://string-functions.com/string-hex.aspx for the test
+    expect(char2Bytes('Taquito is awesome!')).toEqual('5461717569746f20697320617765736f6d6521');
+  });
+
+  it('Should convert bytes to string', () => {
+    expect(bytes2Char('5461717569746f20697320617765736f6d6521')).toEqual('Taquito is awesome!');
+  });
+});
```

Comment:
Yes, I think maybe making the functions in tzip16 deprecated, and update them to actual depend on the utils methods would be the way to go. We can then remove these methods in `8.1`.

Arguably, and I'm interested in your opinion @roxaneletourneau, tzip16 has not been released as `latest` release, only as pre-releases, so we could just move them. Some users of the prerelease's will have to update. This could be acceptable, and we avoid carrying the tech-debt for a couple releases.

Reply:
> Arguably, and I'm interested in your opinion @roxaneletourneau, tzip16 has not been released as latest release, only as pre-releases, so we could just move them. Some users of the prerelease's will have to update. This could be acceptable, and we avoid carrying the tech-debt for a couple releases.

Yes, as tzip16 was a pre-release, I agree on just moving the functions and write a note about that in the next release.